src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const images = document.querySelectorAll('.list-item');
+const bigImage = document.getElementById('largeImg');
+
+images.forEach(image => {
+  image.addEventListener('click', (el) => {
```

Comment:
```suggestion
  image.addEventListener('click', (e) => {
```
it's an event object. `el` is a wrong name.